docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
<p>Do you really mean "self-join" or do you mean "repeated"? In matrix profiles, "self-join" has a specific meaning so we don't want to overload the that term. Also, why repeat it 10 times? Why isn't 3 times sufficient?</p><p> </p><p>Also, instead of</p><p> </p><pre class="ql-syntax" spellcheck="false">base = np.concatenate([motif] * 10)&nbsp;
</pre><p> </p><p>Maybe you can do:</p><p> </p><pre class="ql-syntax" spellcheck="false">base = np.tile(motif, 10)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I am not sure whether self-join means self-concatenation for time series but I will change it to repeated just to disambiguate with the way they use it in matrix profile context. Changed it to tile as well. 10 is unnecessarily high probably - switched to 50 instead and remove the axis limits</p>